crates/base_db/src/input.rs

```diff
@@ -192,6 +192,7 @@ pub struct ProcMacro {
 pub struct CrateData {
     pub root_file_id: FileId,
     pub edition: Edition,
+    pub version: Option<semver::Version>,
```

Comment:
Do we need this to be `semver::Version` or can it just be a regular string?

Reply:
Probably not, I already didn't expose the crate from hir due to asking myself the same question so this can probably also just be a plain String instead.